support/cas-server-support-dynamodb-ticket-registry/src/test/java/org/apereo/cas/ticket/registry/DynamoDbTicketRegistryFacilitatorBillingModePayPerRequestTests.java

```diff
@@ -0,0 +1,102 @@
+import org.apereo.cas.config.CasCoreWebConfiguration;
+import org.apereo.cas.config.CasPersonDirectoryConfiguration;
+import org.apereo.cas.config.DynamoDbTicketRegistryConfiguration;
+import org.apereo.cas.config.DynamoDbTicketRegistryTicketCatalogConfiguration;
+import org.apereo.cas.config.support.CasWebApplicationServiceFactoryConfiguration;
+import org.apereo.cas.logout.config.CasCoreLogoutConfiguration;
+import org.apereo.cas.util.junit.EnabledIfPortOpen;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.autoconfigure.RefreshAutoConfiguration;
+import software.amazon.awssdk.core.SdkSystemSetting;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.BillingMode;
+import software.amazon.awssdk.services.dynamodb.model.DescribeTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.DescribeTableResponse;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * This is {@link DynamoDbTicketRegistryFacilitatorBillingModePayPerRequestTests}.
+ *
+ * @author Ned Regina
+ *
+ */
+@EnabledIfPortOpen(port = 8000)
+@SpringBootTest(classes = {
+        DynamoDbTicketRegistryConfiguration.class,
+        DynamoDbTicketRegistryTicketCatalogConfiguration.class,
+        CasCoreTicketsConfiguration.class,
+        CasCoreTicketCatalogConfiguration.class,
+        CasCoreLogoutConfiguration.class,
+        CasCoreHttpConfiguration.class,
+        CasCoreServicesConfiguration.class,
+        CasCoreAuthenticationConfiguration.class,
+        CasCoreTicketIdGeneratorsConfiguration.class,
+        CasCoreServicesAuthenticationConfiguration.class,
+        CasCoreConfiguration.class,
+        CasCoreNotificationsConfiguration.class,
+        CasCoreWebConfiguration.class,
+        CasCoreUtilConfiguration.class,
+        CasWebApplicationServiceFactoryConfiguration.class,
+        CasCoreAuthenticationServiceSelectionStrategyConfiguration.class,
+        CasCoreAuthenticationHandlersConfiguration.class,
+        CasCoreAuthenticationMetadataConfiguration.class,
+        CasCoreAuthenticationPolicyConfiguration.class,
+        CasCoreAuthenticationPrincipalConfiguration.class,
+        CasCoreAuthenticationSupportConfiguration.class,
+        CasPersonDirectoryConfiguration.class,
+        RefreshAutoConfiguration.class
+},
+        properties = {
+                "cas.ticket.registry.dynamo-db.endpoint=http://localhost:8000",
+                "cas.ticket.registry.dynamo-db.drop-tables-on-startup=true",
+                "cas.ticket.registry.dynamo-db.local-instance=true",
+                "cas.ticket.registry.dynamo-db.region=us-east-1",
+                "cas.ticket.registry.dynamo-db.billingMode=PAY_PER_REQUEST"
```

Comment:
Should be `billing-mode`. Please keep the naming strategy consistent.

Reply:
Will do.